addons/l10n_ec_website_sale/views/website_sales_templates.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="partner_info" name="Ecuadorian partner">
+
+        <!-- show identification type -->
+        <div t-attf-class="mb-3 #{error.get('l10n_latam_identification_type_id') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label" for="l10n_latam_identification_type_id">Identification Type</label>
+            <t t-if="partner.can_edit_vat()">
+                <select name="l10n_latam_identification_type_id" t-attf-class="form-select #{error.get('l10n_latam_identification_type_id') and 'is-invalid' or ''}">
+                    <option value="">Identification Type...</option>
+                    <t t-foreach="identification_types or []" t-as="id_type">
+                        <option t-att-value="id_type.id" t-att-selected="id_type.id == int(identification) if identification else id_type.id == partner.l10n_latam_identification_type_id.id">
+                            <t t-esc="id_type.name"/>
+                        </option>
+                    </t>
+                </select>
+            </t>
+            <t t-else="">
+                <p class="form-control" t-esc="partner.l10n_latam_identification_type_id.name" readonly="1" title="Changing Identification type is not allowed once document(s) have been issued for your account. Please contact us directly for this operation."/>
+                <input name="l10n_latam_identification_type_id" class="form-control" t-att-value="partner.l10n_latam_identification_type_id.id" type='hidden'/>
+            </t>
+        </div>
+
+    </template>
+
+    <template id="address" inherit_id="website_sale.address">
+        <!-- Use a whole row for the Company Name to put Identifaction Type and Identifaction Number (vat) on the same line -->
+        <xpath expr="//input[@name='company_name']/.." position="attributes">
+            <t t-if="res_company.country_id.code == 'EC'">
+                <attribute name="t-attf-class">#{error.get('company_name') and 'o_has_error'} col-lg-12 mb-2</attribute>
+            </t>
+        </xpath>
+
+        <xpath expr="//input[@name='vat']/.." position="before">
+            <t t-if="mode[1] == 'billing'">
+                <t t-if="res_company.country_id.code == 'EC'">
+                    <t t-set="partner" t-value="website_sale_order.partner_id"/>
+                    <t t-call="l10n_ec_website_sale.partner_info"/>
+                </t>
+            </t>
+        </xpath>
+
+        <label for="vat" position="attributes">
+            <attribute name="t-if">res_company.country_id.code != 'EC'</attribute>
```

Comment:
@svfu-odoo I have already commented something about this on the Peru PR and got this reaction: https://github.com/odoo/odoo/pull/130155#discussion_r1422188386

Reply:
@jco-odoo  I commented on the other linked PR.
I also did some other changes so AR and EC website can both be installed at the same without issues (and I removed the demo payment due to remove the (implicit) dependency on it.
https://github.com/odoo/odoo/compare/a766a5405eb9db90d94be927f3489861c7a2befc..b207907f7f69c0f798e8c88c89bb3a3116985e95